### DISCUSSION TITLE
力積電去年營收656億破紀錄 攜手滿拓強攻AI新商機

### DISCUSSION THREAD

**Discussion by jacky0115** [1101-GS3073-C/Python-Community-GS3073-C#185] [科技新知]
[力積電去年營收656億破紀錄 攜手滿拓強攻AI新商機](https://udn.com/news/story/7240/6023463)
力積電是全球唯一同時擁有記憶體、邏輯二大製程技術的專業晶圓代工廠，目前正致力於將邏輯電路與DRAM整合到單一晶片，以 AIM (AI Memory) 技術概念來突破傳統馮紐曼結構瓶頸，解決目前DSP /GPU架構因記憶體傳輸頻寬不足，運算速度受限、高耗能的困境。
以行車安全最需要的 ADAS（先進駕駛輔助系統）應用為例，若是採用AIM技術所開發的單晶片，搭配影像神經網路加速運算程式（Video Neuron Network Accelerator，VNNA），在進行1080p RGB 影像物件分離處理上，可以達到10倍的執行效率，但耗電量卻只要原來的十分之一。
所謂的 AI 晶片（AI Memory）技術，是將 MCU 與 DRAM 整合到單一晶片上，並讓記憶體資料可依照需求進行非循序存取，進而提升影像神經網路運算 10 倍處理效率 !

**Comment by wh20502** [comment]
感覺是要搶占AI晶片製成的市場，但台積電可能還是大宗。
